Src/java/cql-to-elm/src/test/java/org/cqframework/cql/cql2elm/IncludedSignatureOutputTests.java

```diff
@@ -42,31 +32,14 @@ private Library getLibrary(LibraryBuilder.SignatureLevel signatureLevel) throws
         return library;
     }
 
+    private static CqlTranslator getTranslator(LibraryBuilder.SignatureLevel signatureLevel) throws IOException {
+        return TestUtils.getTranslator(CQL_TEST_FILE, LIBRARY_SOURCE_PROVIDER, signatureLevel);
+    }
+
     @Test
     public void TestNone() throws IOException {
-        Library library = getLibrary(LibraryBuilder.SignatureLevel.None);
-
-        // Verify none of the outputs have signatures
-        ExpressionDef def = defs.get("TestOverload");
-        assertThat(((FunctionRef)def.getExpression()).getSignature().size(), is(0));
-
-        def = defs.get("TestOverloadOneInt");
-        assertThat(((FunctionRef)def.getExpression()).getSignature().size(), is(0));
-
-        def = defs.get("TestOverloadOneDecimal");
-        assertThat(((FunctionRef)def.getExpression()).getSignature().size(), is(0));
-
-        def = defs.get("TestOverloadTwoInts");
-        assertThat(((FunctionRef)def.getExpression()).getSignature().size(), is(0));
-
-        def = defs.get("TestOverloadTwoDecimals");
-        assertThat(((FunctionRef)def.getExpression()).getSignature().size(), is(0));
-
-        def = defs.get("TestOverloadOneIntOneDecimal");
-        assertThat(((FunctionRef)def.getExpression()).getSignature().size(), is(0));
-
-        def = defs.get("TestOverloadOneIntTwoDecimal");
-        assertThat(((FunctionRef)def.getExpression()).getSignature().size(), is(0));
+        final CqlTranslator translator = getTranslator(LibraryBuilder.SignatureLevel.None);
+        assertThat(translator.getErrors().size(), is(8));
```

Comment:
Why the error count 8 matters! or we just can consider if specific error is in the list!

Reply:
Thanks.  I've made the changes you suggested to this test and 2 other similar tests.